### PR TITLE
perf(issues): add skeleton UI for issue list loading

### PR DESCRIPTION
**Pull request by JonasBa**
Add skeleton loading UI for the issues stream page. We only get rid of the first initial loader, the rest is as-is today. Should also help with CLS score on this page

### CODE REVIEW

File: static/app/components/stream/group.tsx [hunk 1]
```diff
@@ -95,8 +93,7 @@ function BaseGroupRow({
   narrowGroups = false,
   showLastTriggered = false,
 }: Props) {
-  const groups = useLegacyStore(GroupStore);
-  const group = groups.find(item => item.id === id) as Group;
```
Comment (JonasBa) [inline static/app/components/stream/group.tsx:99]: All callers already resolve the group via GroupStore.get(), they just weren't passing it down to the react component

**Comment by JonasBa** [comment]
Ok, seems like this.state.issuesLoading never changes in some of these tests :wat:

File: static/app/views/issueList/groupListBody.tsx [hunk 2]
```diff
@@ -108,15 +106,23 @@ function GroupList({
 
   return (
     <PanelBody>
-      {groupIds.map((id, index) => {
+      {(loading ? new Array(10).fill(0) : groupIds).map((id, index) => {
```
Comment (malwilley) [inline static/app/views/issueList/groupListBody.tsx:109]: Maybe make this 25 to match the limit we send in the request?

**Review by malwilley — COMMENTED** [review]
This is great. I've been wanting to do this for a long time, thanks for taking the initiative 💯

File: static/app/views/issueList/groupListBody.tsx (hunk 2 above)
Comment (JonasBa) [reply static/app/views/issueList/groupListBody.tsx:109]: 👍🏼

**Review by scttcper — APPROVED** [review]
Nice!

**Review by scttcper — APPROVED** [review]
looks good

**Review by malwilley — APPROVED** [review]
Not seeing any bugs here - looks really nice!

**Comment by JonasBa** [comment]
Egh, double checking this and I found a race condition. Because I removed the useLegacyStore to directly pass groups to the stream component and because we only pass ids to the list, it means that there is a chance we miss the reflux trigger and end up with a partial store... I guess that's why the old code was always calling useLegacyStore...

**Pull request closed without merging**